### DISCUSSION TITLE
svg image wrapper

### DISCUSSION THREAD

**Discussion by arunvenkat1911** [microsoft/react-native-windows#9705] [Q&A] [answered]
Hi Team, 

Do we have image wrapper for svg image in native windows.
can you guide me how to use? package name? 
"react-native-svg-uri" does not seem to work.

Regards, 
Arun.

**Comment by chrisglein** [comment, marked as answer]
There's an active PR that implements react-native-svg that you can find out about [here](https://github.com/microsoft/react-native-windows/issues/3326). Also you can pass SVG files to the `Image` component directly.
